src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestStage.java

```diff
@@ -201,12 +209,6 @@ private void putRemote(String fullFilePath, byte[] data, int retryCount)
             e.getMessage());
         throw new SFException(e, ErrorCode.IO_ERROR);
       }
-
-      if (isCredentialsExpiredException(e)) {
```

Comment:
I might miss something, but this function is not used anymore?

Reply:
yes, as discussed in slack, we will attempt to refresh token on first exception. so this method is not needed anymore.

Reply:
Then the entire `isCredentialsExpiredException` function can be removed? Basically we say we always refresh the credentials on the first retry

Reply:
yess. its removed. I might miss something but I am confused. 
`refreshSnowflakeMetadata` is called above

Reply:
I realised i removed it but only from my mind, will remove the function here. thanks for calling it out!